LIQUIBASE/changelog/v5.1.2/db-changelog-create-view-5.1.2-postgres.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+
+    <changeSet author="nomikosi" id="Alter uuids for db tables">
+
+
+    </changeSet>
+
+    <changeSet id="change_moveconn_value_to_string" author="nomikosi">
+        <modifyDataType tableName="movementconnect" columnName="moveconn_value" newDataType="VARCHAR(36)"/>
+        <rollback>
+            <modifyDataType tableName="movementconnect" columnName="moveconn_value" newDataType="UUID"/>
+        </rollback>
+    </changeSet>
+    <changeSet id="change_move_duplicate_id_to_string" author="nomikosi">
+        <modifyDataType tableName="movement" columnName="move_duplicate_id" newDataType="VARCHAR(36)"/>
+        <rollback>
+            <modifyDataType tableName="movement" columnName="move_duplicate_id" newDataType="UUID"/>
+        </rollback>
+    </changeSet>
+    <changeSet id="change_move_guid_to_string" author="nomikosi">
+        <dropView viewName="LATEST_POS_VIEW"/>
+        <modifyDataType tableName="movement" columnName="move_guid" newDataType="VARCHAR(36)"/>
+        <createView viewName="LATEST_POS_VIEW">
+            SELECT
+            b.move_guid, b.move_heading, b.move_location, b.move_moveconn_id, b.move_timestamp, b.move_speed
+            FROM movement.movement b
```

Comment:
Since we have the opportunity, we can correct this old error: we already specify a default schema in Liquibase. Do NOT use the schema prefixes, i.e. use `FROM movement b`